api/pkg/shared/resource/resource.go

```diff
@@ -47,7 +48,7 @@ var (
 
 // Query resources based on name, kind, tags.
 // Match is the type of search: 'exact' or 'contains'
-// Fields: name, []kinds,[]Catalogs []Tags, Match, Limit
+// Fields: name, []kinds,[]Catalogs []Tags, []Platforms, Match, Limit
```

Comment:
could you add a test for querying using `platforms`?

Reply:
done